.github/workflows/summarize-test.yml

```diff
@@ -111,7 +119,7 @@ jobs:
 
       - name: Download models
         run: |
-          ./occ llm:download-model llama-2
+          ./occ llm:download-model ${{ matrix.model }}
```

Comment:
It's best to also put the matrix value in the title of the job to easily distinguish the different jobs